include/freerdp/codec/h264.h

```diff
@@ -46,7 +46,10 @@ extern "C"
 		H264_CONTEXT_OPTION_RATECONTROL,
 		H264_CONTEXT_OPTION_BITRATE,
 		H264_CONTEXT_OPTION_FRAMERATE,
-		H264_CONTEXT_OPTION_QP
+		H264_CONTEXT_OPTION_QP,
+#ifdef WITH_VAAPI
```

Comment:
would change this to `H264_CONTEXT_OPTION_ACCELERATED` without the `#ifdef` guards to have a generic `I want hardware acceleration if available` option